### PR TITLE
[GraFx Studio] Autogrow

### PR DESCRIPTION
**Pull request by BramVerniest**

### CODE REVIEW

File: docs/GraFx-Studio/guides/text-frame/errorhandling.png
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/errorhandling.png:1]: The "Fail output" dropdown menu is not available yet.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 1]
```diff
@@ -1,55 +1,97 @@
...
+
+![screenshot](autoresize.png)
+
+#### Text Overflow
+
+With no resizing option selected, text that exceeds the frame will trigger a yellow text-overflow indicator below the frame.
+
+![screenshot](overflow.png)
+
+See also [Output Settings](/GraFx-Studio/guides/output/settings/) to define how a VDP job should handle text overflow.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:59]: You can only configure "Continue batch output after failure" for now. Text overflow is a failure.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 2]
```diff
@@ -1,55 +1,97 @@
...
+
+![screenshot](errorhandling.png)
+
+#### Copyfitting
+
+Select the text frame and enable "Copyfitting" to adjust font size to fit the frame by allowing slight reduction or growth.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:65]: Not only a slight reduction or growth. That depends on your min/max settings.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 3]
```diff
@@ -1,55 +1,97 @@
...
 ![screenshot](copyfit.png)
 
-Choose the minimum and maximum percentage a font size can vary.
+Set the minimum and maximum percentages for font size adjustments.
+
+> Note: When Copyfitting is enabled, Autogrow is disabled.
+
+#### Auto-grow
+
+Enable "Autogrow" to allow the frame to expand as needed to fit content of varying lengths. For instance, records in a dataset might contain differing character counts.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:75]: Expand or shrink is also possible.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 4]
```diff
@@ -1,55 +1,97 @@
...
+#### Copyfitting
+
+Select the text frame and enable "Copyfitting" to adjust font size to fit the frame by allowing slight reduction or growth.
 
 ![screenshot](copyfit.png)
 
-Choose the minimum and maximum percentage a font size can vary.
+Set the minimum and maximum percentages for font size adjustments.
+
+> Note: When Copyfitting is enabled, Autogrow is disabled.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:71]: I would also mention this for auto-grow. When that is enabled, copyfitting is disabled. They cannot be enabled together.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 5]
```diff
@@ -1,55 +1,97 @@
...
+
+![screenshot](autogrow-left.png)
+
+When selecting "Left" or "Right," you can also add the opposite direction for bidirectional growth.
+
+![screenshot](autogrow-left-right.png)
+
+When "Up" is selected, you can also add "Down."
+
+To enable Autogrow, select at least one growth direction, or uncheck "Autogrow" to disable it.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:87]: To enable it you have to use the toggle. The component with the arrows is only visible when it's enabled, so you cannot use it to enable it.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 6]
```diff
@@ -1,55 +1,97 @@
...
+Select the text frame and enable "Copyfitting" to adjust font size to fit the frame by allowing slight reduction or growth.
 
 ![screenshot](copyfit.png)
 
-Choose the minimum and maximum percentage a font size can vary.
+Set the minimum and maximum percentages for font size adjustments.
+
+> Note: When Copyfitting is enabled, Autogrow is disabled.
+
+#### Auto-grow
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:73]: Maybe also good to mention that auto-grow cannot be used when the frame is rotated. When the rotation is different from 0º, auto-grow cannot be enabled. Or when auto-grow is enabled, the frame cannot be rotated anymore.

**Review by jeroencosyn — CHANGES_REQUESTED** [review]

**Comment by BramVerniest** [comment]
Will re-open. Need PR slot

**Pull request closed without merging**